cedar-wasm-example/__tests__/main.test.ts

```diff
@@ -105,20 +105,18 @@ describe('Authorizer tests', () => {
             resource: { type: 'App::Code', id: 'this' },
             context: {},
             schema: {
-                json: {
-                    App: {
```

Comment:
Huh, I thought you did need the `json:` tag now.  How do you pass a human-syntax schema in TS?